specs/DesignDiscussions/InputLabels.md

```diff
@@ -91,6 +129,42 @@ If we want to allow the host to control formatting of the labels, either in gene
 ```
 Another option is to not put this in host config, and instead control this styling via support of native styling on required and optional labels (i.e. CSS, Xaml Styles, etc). This decision speaks to a larger discussion of the direction of host config that is beyond the scope of this document.
```

Comment:
Is this ok to be noted as option we considered but discarded owing to the fact that it's out of scope here?